tests/functional/api/test_groups.py

```diff
@@ -44,3 +59,12 @@ def group_member_with_token(group_member, db_session, factories):
     db_session.add(token)
     db_session.commit()
     return (group_member, token)
+
+
+@pytest.fixture
+def user_with_token(db_session, factories):
+    user = factories.User()
+    token = factories.DeveloperToken(userid=user.userid)
+    db_session.add(token)
+    db_session.commit()
```

Comment:
I _think_ the `add()` here is unnecessary (`user` will have been added automatically by the factory, and `token` will have been added because it's attached to `user`).